internals/cli/cli.go

```diff
@@ -139,16 +139,16 @@ func fixupArg(optName string) string {
 	return optName
 }
 
-type clientSetter interface {
-	setClient(*client.Client)
+type ClientInitializer interface {
+	Init(*client.Client)
```

Comment:
Name bike-shedding, but:

Should this be called `InitClient`? I know it's sort of redundant as Go will only match the method if the types match as well, but `Init` seems too generic: is it initing the command, the client, or something else? Cf: before it was `setClient`, not `set`.

Then again, it seems this is really setting the client, not initializing it. So why not `ClientSetter` / `SetClient` more like it was?

Reply:
We had a quick conversation about this. We will take this as is to Gustavo for now. The ```Init()``` choice does not immediately make sense in the context of Pebble, but the external project extending the client will actually have an ```Init()``` that does more, creating a new extended client instance. So its more that ```Init()``` now allows you to create a new extended client, but you do not have to use this facility if you are happy with the default Pebble client instance.